Release v2.0.0

Tasshack opened the pull request.

Tasshack committed:
Merge pull request #189 from Tasshack/master

Update FUNDING.yml

Commit:
v2.0.0

Commit:
#190

Commit:
v2.0.0b2

Commit:
v2.0.0b3

Commit:
v2.0.0b4

Commit:
v2.0.0b4

Commit:
v2.0.0b5

Commit:
v2.0.0b5

WZYProjects committed:
Portuguese translation v2.0.0

pt-PT and pt-BR tested on v2.0.0b5

Tasshack committed:
Merge pull request #215 from WZYProjects/dev

Portuguese translation v2.0.0

Commit:
v2.0.0b6

Commit:
Merged from master

pablonube commented:
Go!!

Commit:
Updated and added DE translations

Fresh translations for German users. Still working out some function bits as the official app has clunky translations as well.

Tasshack committed:
Merge pull request #274 from 13robin37/dev

Updated and added DE translations

Commit:
v2.0.0b7

Commit:
v2.0.0b8

Commit:
v2.0.0b8

v1k70rk4 committed:
Create hu.json

Preparing Hungarian translation of the current en.json (v2.0.0b7) into Hungarian.

Tasshack committed:
Merge pull request #306 from v1k70rk4/dev

Create hu.json

Commit:
v2.0.0b9

gentslava committed:
Ru localization improvements

Commit:
v2.0.0b10

Commit:
v2.0.0b11

gentslava committed:
Merge branch 'dev' into dev

Tasshack committed:
Merge pull request #326 from gentslava/dev

Russian localization (ru)

Commit:
Added a Dutch translation file. Added PyCharm .idea direcotry to ignore file.

Tasshack committed:
Merge pull request #360 from yodax/dutch-translation

Dutch translation

kmsg1820 commented:
I request an update of m30 pro

stefan5300 commented:
When will you be merging it into master?When will you be able to merge this into master?
Was trying to update the module via HACS to support the dreame login but it's not in master yet :)

kkuriata commented:
What about this release?

Commit:
v2.0.0b12

Commit:
v2.0.0b12

Commit:
v2.0.0b12

Commit:
v2.0.0b12

Commit:
v2.0.0b12

Tasshack committed:
Update README.md

Commit:
v2.0.0b12

pablonube commented:
Merge please!

yarikoptic committed:
Add rudimentary codespell config

yarikoptic committed:
Add github action to codespell master on push and PRs

yarikoptic committed:
Add custom codespell skips

Commit:
v2.0.0b13

yarikoptic committed:
[DATALAD RUNCMD] run codespell throughout fixing typos automagically

=== Do not change lines below ===
{
 "chain": [],
 "cmd": "codespell -w",
 "exit": 0,
 "extra_inputs": [],
 "inputs": [],
 "outputs": [],
 "pwd": "."
}
^^^ Do not change lines above ^^^

Tasshack committed:
Merge pull request #471 from yarikoptic/enh-codespell

Add codespell and fix typos

Commit:
v2.0.0b14

Commit:
v2.0.0b14

swiftbird07 committed:
Fixed German translation

Tasshack committed:
Merge pull request #488 from swiftbird07/patch-1

Refined German Translations

AdriVillaB committed:
Second version of the spanish translation, updating to the one in dev branch

Tasshack committed:
Merge pull request #509 from AdriVillaB/dev

Second version of the spanish translation, updating to the one in dev.

Commit:
v2.0.0b15

Commit:
Merge branch 'dev' of https://github.com/Tasshack/dreame-vacuum into dev

Commit:
v2.0.0b15

Commit:
v2.0.0b15

Commit:
fix: #498 Support for new Xiaomi Robot Vacuum X20+

Commit:
v2.0.0b16

Commit:
v2.0.0b16

Commit:
v2.0.0b16

Commit:
v2.0.0b16

dtalens committed:
New translation (Catalan)

Tasshack committed:
Merge pull request #681 from dtalens/dev

New translation (Catalan)

xangin committed:
New translation (Taiwanese)

Add Taiwanese translation.

xangin committed:
Update zh-Hant.json

Tasshack committed:
Merge pull request #730 from xangin/patch-1

New translation (Taiwanese)

taczirjak committed:
Update hu.json

Tasshack committed:
Merge pull request #764 from taczirjak/patch-2

Update hu.json

maodun96 committed:
Create zh-Hans.json

Submit the Simplified Chinese translation file, quickly modified from the zh-Hant.json file.

taifunk committed:
v2.0.0b16
Added lines to resolve error
https://github.com/Tasshack/dreame-vacuum/issues/778

Tasshack committed:
Merge pull request #789 from maodun96/patch-1

added zh-Hans translations

Tasshack committed:
Update FUNDING.yml

suskozaver committed:
Added Slovenian translations

suskozaver committed:
Slovenian translation

suskozaver committed:
Slovenian translations

suskozaver committed:
Slovenian translation

Tasshack committed:
Merge branch 'dev' into Slovenian-translation

Tasshack committed:
Update .gitignore

rubentsirunyan committed:
Checking if history.neglected_segments is a dict

Commit:
Fix translation key for fan speed error in German locale

Updated the key from "fan_speed" to "fan_speed_error" in the German translation file to ensure consistency and accuracy. No other changes were made to the file.

Tasshack committed:
Merge pull request #872 from kagithd/dev

Fix translation key for fan speed error in German locale

Tasshack committed:
Merge pull request #844 from suskozaver/Slovenian-translation

Slovenian translation

Tasshack committed:
Merge pull request #805 from taifunk/master

v2.0.0b16

Tasshack committed:
Merge pull request #867 from rubentsirunyan/dev

Checking if history.neglected_segments is a dict

Tasshack committed:
Update and rename hacs.yaml to validate.yaml

Tasshack committed:
Update validate.yaml

Commit:
Merge branch 'master' into dev

Commit:
v2.0.0b17

Commit:
v2.0.0b17

Commit:
v2.0.0b17

ThibaultDenoual committed:
Corrected fr.json

Drafteed committed:
Update ru translations

Drafteed committed:
tweaks

GuiGu0101 committed:
fix: Crash caused when device.status.deodorizer_life is None

ematt committed:
Add Romanian

Tasshack committed:
Merge pull request #904 from ThibaultDenoual/dev

Corrected fr.json

Tasshack committed:
Merge pull request #933 from Drafteed/translations-ru-fixes

Update ru translations

Tasshack committed:
Merge pull request #963 from ematt/ro_translation

Add Romanian

Tasshack committed:
Merge pull request #941 from GuiGu0101/dev

fix: Crash caused when device.status.deodorizer_life is None

Kan55 committed:
Add Czech translation

Kan55 committed:
fixed JSON formating

Tasshack committed:
Merge pull request #999 from Kan55/czech_translation

Czech translation

Commit:
v2.0.0b18

Commit:
v2.0.0b18

Commit:
v2.0.0b18

Commit:
v2.0.0b18

Commit:
v2.0.0b18

Commit:
Update README.md

Commit:
v2.0.0b18

Commit:
Merge branch 'master' into dev

Commit:
v2.0.0b19

LOVE2CMOL committed:
Update Chinese translations

Tasshack committed:
Merge pull request #1074 from LOVE2CMOL/translation_zhHans

Update zh-Hans translations

hyeok-yoo committed:
Added Korean translation

Tasshack committed:
Merge pull request #1126 from shyoo0000/dev

Added Korean translation

detecti1 committed:
add as_dict for some type, avoid serialization error when using ltss

Tasshack committed:
Merge pull request #1135 from detecti1/fix/data-json-serialization

Fix JSON serialization error while using LTSS

Commit:
v2.0.0b20

Commit:
v2.0.0b20

Commit:
v2.0.0a21 with Trouver Support

Commit:
Improve Dutch translations in nl.json

evensure committed:
fixup! v2.0.0b20

evensure committed:
fixup! v2.0.0b20

mailforMike committed:
Update verification path in verify_code method

Chreece committed:
Create el.json

Chreece committed:
Update el.json

Added support for greek language

Tasshack committed:
Merge pull request #1192 from Tasshack/trouver_support

Trouver Support

Tasshack committed:
Merge pull request #1225 from mailforMike/dev

Update verification path in verify_code method

Tasshack committed:
Merge pull request #1220 from evensure/dev

Fixed 1 English typo and 2 German translations

Tasshack committed:
Merge pull request #1200 from KevinKerkhofs/improve-nl-translation

Improve Dutch translations in nl.json

Tasshack committed:
Merge pull request #1227 from Chreece/dev

Create el.json

Commit:
Fix German translation conflict

Commit:
Fix German translation conflict

Commit:
v2.0.0b21

Drafteed committed:
Fixes of migrate integration to has entity name

arturpragacz committed:
Implement clean area

Tasshack committed:
Merge pull request #1330 from Drafteed/migrate-entity-naming2

Fixes of migrate integration to has entity name

Commit:
v2.0.0b21

Commit:
v2.0.0b21

Commit:
v2.0.0b22

Commit:
Merge branch 'dev' of https://github.com/Tasshack/dreame-vacuum into dev

Commit:
v2.0.0b22

Commit:
v2.0.0b22

Commit:
v2.0.0b22

djremco committed:
Revise Dutch translations in nl.json

Updated Dutch translations for various vacuum states and modes.

djremco committed:
Update Dutch translations for vacuum component

djremco committed:
Update translation for already_configured message

z-master42 committed:
Add device_class to various sensor descriptions

z-master42 committed:
Correct units of measurement

arturpragacz committed:
Backwards compatibility for older HA versions

Tasshack committed:
Merge pull request #1476 from arturpragacz/clean_area

Implement clean area

Tasshack committed:
Merge pull request #1508 from djremco/dev

Updated Dutch Translation

Tasshack committed:
Merge pull request #1542 from z-master42/patch-2

Correct units of measurement

Tasshack committed:
Merge pull request #1543 from z-master42/patch-1

Add device_class to various sensor descriptions

Commit:
v2.0.0b23

Commit:
v2.0.0b23

Commit:
v2.0.0b23

Commit:
v2.0.0b23

Commit:
v2.0.0b24

Commit:
v2.0.0b25
